packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -0,0 +1,606 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+import {
+  arraysAreEqual,
+  GraphPageIterator,
+  MgtTemplatedComponent,
+  Providers,
+  ProviderState
+} from '@microsoft/mgt-element';
+import * as MicrosoftGraph from '@microsoft/microsoft-graph-types';
+import { customElement, html, internalProperty, property, TemplateResult } from 'lit-element';
+import { repeat } from 'lit-html/directives/repeat';
+import {
+  getDriveFilesByIdIterator,
+  getDriveFilesByPathIterator,
+  getFilesByIdIterator,
+  getFilesByListQueryIterator,
+  getFilesByPathIterator,
+  getFilesByQueries,
+  getFilesIterator,
+  getGroupFilesByIdIterator,
+  getGroupFilesByPathIterator,
+  getMyInsightsFiles,
+  getSiteFilesByIdIterator,
+  getSiteFilesByPathIterator,
+  getUserFilesByIdIterator,
+  getUserFilesByPathIterator,
+  getUserInsightsFiles
+} from '../../graph/graph.files';
+import '../sub-components/mgt-spinner/mgt-spinner';
+import { OfficeGraphInsightString, ViewType } from '../../graph/types';
+import { styles } from './mgt-file-list-css';
+
+/**
+ * The File List component displays a list of multiple folders and files by
+ * using the file/folder name, an icon, and other properties specicified by the developer.
+ * This component uses the mgt-file component.
+ *
+ * @export
+ * @class MgtFileList
+ * @extends {MgtTemplatedComponent}
+ *
+ * @cssprop
```

Comment:
missing `@cssprop`s and `@fires` for events

Reply:
Good catch